packages/browser-ui/src/inspector/views/Opt.tsx

```diff
@@ -1,8 +1,15 @@
 import * as React from "react";
 import IViewProps from "./IViewProps";
 import { prettyPrintFn, evalFns } from "@penrose/core";
-import { zip } from "lodash";
+import { zipWith, sum } from "lodash";
 import DataTable from "react-data-table-component";
+
+export const EPS = 10e-3;
+
+const vnorm = (v: number[]): number => {
```

Comment:
There is a `vnorm` already in `ops` in some Utils file that you could import, if you like.